internal/provider/features.go

```diff
@@ -10,6 +10,24 @@ func schemaFeatures() *pluginsdk.Schema {
 	// NOTE: if there's only one nested field these want to be Required (since there's no point
 	//       specifying the block otherwise) - however for 2+ they should be optional
 	featuresMap := map[string]*pluginsdk.Schema{
+		// NOTE: soft delete feature not yet supported with ASH 2108
+		// "key_vault": {
+		// 	Type:     pluginsdk.TypeList,
+		// 	Optional: true,
+		// 	MaxItems: 1,
+		// 	Elem: &pluginsdk.Resource{
+		// 		Schema: map[string]*schema.Schema{
+		// 			"recover_soft_deleted_key_vaults": {
+		// 				Type:     pluginsdk.TypeBool,
+		// 				Optional: true,
+		// 			},
+		// 			"purge_soft_delete_on_destroy": {
+		// 				Type:     pluginsdk.TypeBool,
+		// 				Optional: true,
+		// 			},
+		// 		},
+		// 	},
+		// },
```

Comment:
ditto for all unsupported stuff

Reply:
https://github.com/hashicorp/terraform-provider-azurestack/pull/151/commits/380ae1ae98407c69f3c3a0ad66a69464ba6b0b2b, removed also